packages/lesswrong/components/sunshineDashboard/SunshineNewUserCommentsList.tsx

```diff
@@ -57,7 +57,7 @@ const SunshineNewUserCommentsList = ({terms, classes, truncated=false}: {
             <SmallSideVote document={comment} collection={Comments}/>
           </span>
         </Link>
-        {!truncated && comment.deleted && <div><MetaInfo>`[Comment deleted${comment.deletedReason ? ` because "${comment.deletedReason}"` : ""}]</MetaInfo></div>}
+        {!truncated && comment.deleted && <div><MetaInfo>{`[Comment deleted${comment.deletedReason ? ` because "${comment.deletedReason}"` : ""}]`}</MetaInfo></div>}
```

Comment:
```suggestion
        {!truncated && comment.deleted && <div><MetaInfo>[Comment deleted${comment.deletedReason ? ` because "${comment.deletedReason}"` : ""}]</MetaInfo></div>}
```